src/api.rs

```diff
@@ -1257,24 +1264,38 @@ impl<T: Pixel> ContextInner<T> {
     // Does not include SEF frames.
     let mut nframes_total = 0;
     while ntus < reservoir_frame_delay {
-      if let Some(fi) = self.frame_invariants.get(&output_frameno) {
+      let output_frameno_in_gop =
+       output_frameno - prev_keyframe_output_frameno;
+      let is_kf = if let
+       Some(fi) = self.frame_invariants.get(&output_frameno) {
         if fi.frame_type == FrameType::KEY {
-          collect_counts(nframes, &mut acc);
           prev_keyframe_input_frameno = fi.input_frameno;
-          prev_keyframe_output_frameno = output_frameno;
-          prev_keyframe_ntus = ntus;
-          prev_keyframe_nframes = nframes_total;
           // We do not currently use forward keyframes, so they should always
-          //  end the current TU.
+          //  end the current TU (thus we always increment ntus below).
           debug_assert!(fi.show_frame);
-          output_frameno += 1;
-          ntus += 1;
-          nframes_total += 1;
-          continue;
+          true
+        }
```

Comment:
Could I nag you to fix the coding style? ^^ Otherwise we could run a pass of rustfmt over the whole file.

Reply:
The coding style is consistent with the rest of the file...

Reply:
I'll pass rustfmt once you land this, the part consistent with this is another patch from Tim :)